test/helpers/modifyMetadataHashes.js

```diff
@@ -22,6 +23,10 @@ export default async function modifyMetadataHashes({
     return
   }
 
+  // If fees are enabled, a Transfer event is fired in addition to the Modified event
+  const expectedLogsLength = feesEnabled ? 2 : 1
+  const logIndex = feesEnabled ? 1 : 0
```

Comment:
This should maybe be something like `modifiedLogIndex` instead of just `logIndex`.